lib/screens/v3_api.ex

```diff
@@ -39,7 +42,10 @@ defmodule Screens.V3Api do
         :not_modified
 
       {:response_success, %{status_code: status_code}} = response ->
-        log_api_error({:bad_response_code, response}, status_code: status_code)
+        {:bad_response_code, _} =
+          log_api_error({:bad_response_code, response}, status_code: status_code)
```

Comment:
Is this binding here because credo / dialyzer complained about not using the return value of the `log_api_error` call? If so, we've had an informal convention of just matching that to `_` in the past. Would suggest doing that here as well if we don't really care about the return value at all, and it looks like we don't.
```suggestion
        _ = log_api_error({:bad_response_code, response}, status_code: status_code)
```